partner_identification/models/res_partner.py

```diff
@@ -163,8 +162,10 @@ def _search_identification(self, field_name, category_code,
         Returns:
             list: Domain to search with.
         """
-
-        return [
-            (field_name, operator, value),
+        id_numbers = self.env['res.partner.id_number'].search([
+            ('name', operator, value),
```

Comment:
I did miss the docblock though - `field_name` isn't a param anymore in this method